lib/fuzion/std/fileio.fz

```diff
@@ -66,4 +66,26 @@ public fuzion.std.fileio is
                       # the length of the internal array representing the path
                       pathLength i32) i64 is intrinsic
 
-  private writeFile(path string, content array<u8>) is intrinsic # NYI
+  # creates a new file using an array of u8 as content and a string as the file path
+  # it overrides the file, if it exists, or creates a new one if it does not exist
```

Comment:
overrides -> overwrites?

Reply:
typo because i used to use @Override in Java